internal/provider/privateservice/resource/internal/update.go

```diff
@@ -1,22 +1,24 @@
 package internal
 
 import (
+	"context"
+
 	"terraform-provider-render/internal/client"
 	"terraform-provider-render/internal/provider/common"
 	"terraform-provider-render/internal/provider/privateservice"
 )
 
-func UpdateServiceRequestFromModel(plan privateservice.PrivateServiceModel, ownerID string) (client.UpdateServiceJSONRequestBody, error) {
+func UpdateServiceRequestFromModel(ctx context.Context, plan privateservice.PrivateServiceModel, ownerID string) (client.UpdateServiceJSONRequestBody, error) {
 	envSpecificDetails, err := common.EnvSpecificDetailsForPATCH(plan.RuntimeSource, plan.StartCommand.ValueStringPointer())
 	if err != nil {
 		return client.UpdateServiceJSONRequestBody{}, err
 	}
 
 	servicePlan := client.PaidPlan(plan.Plan.ValueString())
 
-	pullRequestPreviewsEnabled := client.PrivateServiceDetailsPATCHPullRequestPreviewsEnabledNo
+	pullRequestPreviewsEnabled := client.PullRequestPreviewsEnabledNo
```

Comment:
double checking: no functional change as a result of these types of changes, but does make it a ton nicer to read going forwards

Reply:
Correct, this is just respecting the rename of a regenerated const.